lib/segment/src/index/hnsw_index/gpu/gpu_level_builder.rs

```diff
@@ -130,9 +130,11 @@ mod tests {
         )
         .unwrap();
 
-        let graph_layers_builder =
+        let mut graph_layers_builder =
```

Comment:
Can you please move `graph_layers_builder.fill_ready_list();` inside `create_graph_layers_builder` to avoid code duplication and to hide details of `GraphLayersBuilder` here?

Reply:
No, I can not. For the very reason this PR exists.